From 271098a126f673e7ffe13516b8e753df27c9d2bc Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Wed, 3 Jul 2024 14:27:17 -0700
Subject: [PATCH] Docs/process/2.Process: Update mm tree URL

The document introduces http://www.ozlabs.org/~akpm/mmotm/ as the place
for mm tree patches.  However it has moved to git.kernel.org.  Update
the URL.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/process/2.Process.rst                    | 2 +-
 Documentation/translations/it_IT/process/2.Process.rst | 2 +-
 Documentation/translations/sp_SP/process/2.Process.rst | 2 +-
 Documentation/translations/zh_CN/process/2.Process.rst | 2 +-
 Documentation/translations/zh_TW/process/2.Process.rst | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/Documentation/process/2.Process.rst b/Documentation/process/2.Process.rst
index 613a01da4717..c2b16a5d96b0 100644
--- a/Documentation/process/2.Process.rst
+++ b/Documentation/process/2.Process.rst
@@ -288,7 +288,7 @@ mainline get there via -mm.
 The current -mm patch is available in the "mmotm" (-mm of the moment)
 directory at:
 
-	https://www.ozlabs.org/~akpm/mmotm/
+	https://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new.git/tree/pc/devel-series
 
 Use of the MMOTM tree is likely to be a frustrating experience, though;
 there is a definite chance that it will not even compile.
diff --git a/Documentation/translations/it_IT/process/2.Process.rst b/Documentation/translations/it_IT/process/2.Process.rst
index 0a62c0f33faf..a19545b93ba4 100644
--- a/Documentation/translations/it_IT/process/2.Process.rst
+++ b/Documentation/translations/it_IT/process/2.Process.rst
@@ -308,7 +308,7 @@ patch andrà nel ramo principale attraverso -mm.
 La patch -mm correnti sono disponibili nella cartella "mmotm" (-mm of
 the moment) all'indirizzo:
 
-      http://www.ozlabs.org/~akpm/mmotm/
+      https://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new.git/tree/pc/devel-series
 
 È molto probabile che l'uso dei sorgenti MMOTM diventi un'esperienza
 frustrante; ci sono buone probabilità che non compili nemmeno.
diff --git a/Documentation/translations/sp_SP/process/2.Process.rst b/Documentation/translations/sp_SP/process/2.Process.rst
index 5993eed71563..d8309a68d177 100644
--- a/Documentation/translations/sp_SP/process/2.Process.rst
+++ b/Documentation/translations/sp_SP/process/2.Process.rst
@@ -316,7 +316,7 @@ a través de -mm.
 El parche -mm actual está disponible en el directorio “mmotm” (-mm
 del momento) en:
 
-	https://www.ozlabs.org/~akpm/mmotm/
+	https://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new.git/tree/pc/devel-series
 
 Sin embargo, es probable que el uso del árbol MMOTM sea una experiencia
 frustrante; existe una posibilidad definitiva de que ni siquiera se
diff --git a/Documentation/translations/zh_CN/process/2.Process.rst b/Documentation/translations/zh_CN/process/2.Process.rst
index e68c9de0f7f8..e0332883bf11 100644
--- a/Documentation/translations/zh_CN/process/2.Process.rst
+++ b/Documentation/translations/zh_CN/process/2.Process.rst
@@ -218,7 +218,7 @@ Next 树
 
 当前-mm 补丁可在“mmotm”（-mm of the moment）目录中找到：
 
-        https://www.ozlabs.org/~akpm/mmotm/
+        https://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new.git/tree/pc/devel-series
 
 然而，使用MMOTM树可能会十分令人头疼；它甚至可能无法编译。
 
diff --git a/Documentation/translations/zh_TW/process/2.Process.rst b/Documentation/translations/zh_TW/process/2.Process.rst
index f45ddba6238f..278897b4bd39 100644
--- a/Documentation/translations/zh_TW/process/2.Process.rst
+++ b/Documentation/translations/zh_TW/process/2.Process.rst
@@ -221,7 +221,7 @@ Next 樹
 
 當前-mm 補丁可在“mmotm”（-mm of the moment）目錄中找到：
 
-        https://www.ozlabs.org/~akpm/mmotm/
+        https://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new.git/tree/pc/devel-series
 
 然而，使用MMOTM樹可能會十分令人頭疼；它甚至可能無法編譯。
 
-- 
2.39.2

